cli/src/pcluster/utils.py

```diff
@@ -36,18 +36,48 @@
 
 LOGGER = logging.getLogger(__name__)
 
+DEFAULT_PARTITION = "aws"
+PARTITION_MAP = {
+    "cn": "aws-cn",
+    "us-gov": "aws-us-gov",
+    "us-iso-": "aws-iso",
+    "us-isob": "aws-iso-b",
```

Comment:
why not `us-isob-`?

Reply:
No specific reason. I agree to use `us-isob-` instead of `us-isob`.